File: TypeScript/URL/get-path-from-url.ts
import * as url from "url"

function main(): void {
    // Get the path from a given url.
    console.log(getPathFromURL("https://www.example.com/assets/images/logo.png"))
}

main()

// Get the path from a given url.
function getPathFromURL(givenURL: string): string | null {
    return new URL(givenURL).pathname
}

File: Python/URL/get_path_from_url.py
import urllib.parse


# Get the path from a given url.
def get_path_from_url(url):
    return urllib.parse.urlparse(url).path


def main():
    # Get the path from a given url.
    print(get_path_from_url("https://www.example.com/example.html"))


main()